### PR TITLE
chore: update uds-package-requirements.md

### PR DESCRIPTION
**Pull request by oates**
Changed the air-gapped verbiage to match the style we use elsewhere

## Description

...

## Checklist before merging

- [ ] ADR proposed if making an architectural change to the repo
- [x] Tests run, docs added or updated as needed

### CODE REVIEW

**Review by jonnyborbs — APPROVED** [review]
awesome, helpful tweak. let's do it

File: docs/uds-packages/requirements/uds-package-requirements.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Bronze packages:
 - [ ] **Must** deploy and operate successfully with Istio injection enabled in the namespace.
 - [ ] **Must** implement Journey testing, covering the basic user flows and features of the application (see [Testing Guidelines](https://github.com/defenseunicorns/uds-common/blob/main/docs/uds-packages/guidelines/testing-guidelines.md))
 - [ ] **Must** implement Upgrade Testing to ensure that the current development package works when deployed over the previously released one. (see [Testing Guidelines](https://github.com/defenseunicorns/uds-common/blob/main/docs/uds-packages/guidelines/testing-guidelines.md))
-- [ ] **Must** be capable of operating within an internet-disconnected (air-gapped) environment
+- [ ] **Must** be capable of operating within an internet-disconnected (airgap) environment
```
Comment (zachariahmiller) [inline docs/uds-packages/requirements/uds-package-requirements.md:72]:
```suggestion
- [ ] **Must** be capable of operating within an internet-disconnected (airgapped) environment
```

**Review by zachariahmiller — CHANGES_REQUESTED** [review]

File: docs/uds-packages/requirements/uds-package-requirements.md (hunk 1 above)
Comment (oates) [reply docs/uds-packages/requirements/uds-package-requirements.md:72]: Designers suggested  "Must be capabile of operating in an airgap (internet-disconnected) environment."

Comment (zachariahmiller) [inline docs/uds-packages/requirements/uds-package-requirements.md:72]:
```suggestion
- [ ] **Must** be capable of operating in an airgap (internet-disconnected) environment
```

**Review by andrewg-xyz — APPROVED** [review]

File: docs/uds-packages/requirements/uds-package-requirements.md (hunk 1 above)
Comment (oates) [reply docs/uds-packages/requirements/uds-package-requirements.md]: changed

**Pull request merged**